>From 21a97f670d8a2e30f5ad9ed93e6fb973dc8a80a7 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Wed, 4 Sep 2013 09:36:20 +0300
Subject: [PATCH 2/3] In debian-ispell.el Do not display extra messages at
 debian-ispell-set-startup-menu
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 support/emacsen/debian-ispell.el |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/support/emacsen/debian-ispell.el b/support/emacsen/debian-ispell.el
index b5e9a59..41e74e1 100644
--- a/support/emacsen/debian-ispell.el
+++ b/support/emacsen/debian-ispell.el
@@ -244,7 +244,7 @@ To be run at `after-init-hook' or at any time if FORCE is given."
 
     (if (and (featurep 'ispell)
 	     (not force))
-	(message "ispell.el is already loaded")
+	nil
       (when (fboundp 'debian-ispell-build-startup-menu)
 	(debian-ispell-build-startup-menu dicts-list)
 	;; (fmakunbound 'debian-ispell-build-startup-menu)
-- 
1.7.10.4

